src/numeric/decision_rules/compat.jl

```diff
@@ -23,9 +25,9 @@ function evaluate(dr::AbstractDecisionRule{EmptyGrid, CartesianGrid{d}, n_x}, z:
     return reinterpret(Float64, out, (n_x,N))'
 end
 
-evaluate(dr::AbstractDecisionRule{EmptyGrid, <:CartesianGrid}, z::AbstractVector) = vec(evaluate(dr,z'))
-evaluate(dr::AbstractDecisionRule{EmptyGrid, <:CartesianGrid}, i::Int, y::Union{AbstractVector,AbstractMatrix}) = evaluate(dr,y) # kind of nonsensical
-evaluate(dr::AbstractDecisionRule{EmptyGrid, <:CartesianGrid}, x::Union{AbstractVector,AbstractMatrix}, y::Union{AbstractVector,AbstractMatrix}) = evaluate(dr,y)
+evaluate(dr::AbstractDecisionRule{EmptyGrid, <:CartesianGrid}, z::Vector) = vec(evaluate(dr,z'))
```

Comment:
The reasoning is that with abstract we can't guarantee that we can reinterpret as ListOfPoints so I thought it would make everythings clearer if we were to use plain old traditional vectors internally.

Reply:
What do you think ?

Reply:
That makes sense to me. If we ever realize we want the version with `AbstractVector` perhaps we add additional methods that convert the input to `Vector` and use the methods you implemented here.